bundles/org.openhab.ui/web/src/components/addons/addon-card.vue

```diff
@@ -143,8 +143,7 @@ export default {
     },
     imageUrl () {
       if (this.addon.imageLink) return this.addon.imageLink.replace(/^\/\//, 'https://')
-      let docsBranch = 'final'
-      if (this.$store.state.runtimeInfo.buildString === 'Release Build') docsBranch = 'final-stable'
+      let docsBranch = 'final-3.4.x'
```

Comment:
The hard-coded branch name is however needed here, as this whole PR is just there to make the already releases 3.4.x version access the archived docs. main is a. unaffected and b. has improved the whole website link and add-on logo code, so that if we ever need to fix the docs version again after a future release, we have to change less code.